pandas/tests/indexes/period/test_period.py

```diff
@@ -697,3 +697,11 @@ def test_join_self(self, how):
         index = period_range('1/1/2000', periods=10)
         joined = index.join(index, how=how)
         assert index is joined
+
+    def test_insert(self):
```

Comment:
in another PR, can come back and move test_insert to datetimelike.py where this can be tested generically (instead of having code *inside* each datetimelike index type)